index.html

```diff
@@ -6,14 +6,98 @@
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
     <!-- Add your HTML markup here -->
+    <header class="header">
+
+        <a href="#">
+            <img src="favicon.ico" 
+            alt="logo favorite icon"
+            class="header_logo">   
+        </a>
+        
+        <nav>
+            <ul class="navigation_list">
+                <li class="navigation_item">
+                    <a class="navigation_link1" href="/index.html">Meet Karma</a>
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">How It Works</a>
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="/store.html">Store</a>
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">Blog</a>
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">Help</a>
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">Login</a>
+                </li>
+            </ul>
+
+        </nav>
+    </header>
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <main >
+
+        <section class="content1">
+            <div class="Introducing">
+                <h1 class="h1Introducing">Introducing Karma</h1>
+                <p class="pIntroducing">Bring WiFi with you, everywhere you go.</p>
+                <button class="btnLearn">Learn More</button>
+            </div>
+        </section>
+
+        <section class="Logos">
```

Comment:
This section needs to be adjusted in such a way that The Image and the text box exist together in a single `div` like below.

```
<div>
  <img src="/img/icon-devices.svg" alt="device" class="icon">
  <h4 class="iconText">Internet for all devices</h4>
</div>
```

Reply:
I did the changes, put the images and the texts in the same div